experimental/examples/imageviewer/iosApp/iosApp.xcodeproj/project.pbxproj

```diff
@@ -331,7 +331,7 @@
 				CODE_SIGN_IDENTITY = "Apple Development";
 				CODE_SIGN_STYLE = Automatic;
 				DEVELOPMENT_ASSET_PATHS = "";
-				DEVELOPMENT_TEAM = "${TEAM_ID}";
+				DEVELOPMENT_TEAM = 3CP3246D24;
```

Comment:
better to revert TEAM_ID and set it only locally.

Reply:
Thank you! Original snippet was restored.